PyInstaller/utils/win32/icon.py

```diff
@@ -21,6 +21,8 @@
 except AttributeError:
     StringTypes = [ type("") ]
 
+from ...compat import win32api
```

Comment:
It seems that at one time pyinstaller supported not having win32api available. This is no longer the case, so I moved the import.